From c4f9a1920d9a6ac23739445d3256cb81ee744999 Mon Sep 17 00:00:00 2001
From: Avery Pennarun <apenwarr@gmail.com>
Date: Sat, 21 Dec 2013 02:36:45 -0500
Subject: [PATCH 1/7] Makefile: add savedefconfig target.

This writes a file called 'defconfig' which contains only the deltas from
the default configuration.  It's suitable for use as an input defconfig file
for later builds.
---
 Makefile.real | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Makefile.real b/Makefile.real
index b010dd7..f60d5ca 100644
--- a/Makefile.real
+++ b/Makefile.real
@@ -32,6 +32,11 @@ usedefconfig:
 	@$(MAKE) -C kconf conf
 	@./kconf/conf --defconfig=defconfig Kconfig
 
+.PHONY: savedefconfig
+savedefconfig:
+	@$(MAKE) -C kconf conf
+	@./kconf/conf --savedefconfig=defconfig Kconfig
+
 defconfig-%::
 	@$(MAKE) -C kconf conf
 	@./kconf/conf --defconfig=defconfigs/$(@:defconfig-%=%) Kconfig
-- 
1.9.1.423.g4596e3a

